src/client/components/app/Transactions/components/TxTokenInput.tsx

```diff
@@ -122,14 +122,44 @@ const TokenIconContainer = styled.div`
   width: 100%;
 `;
 
+const ZappableTokenButton = styled(Button)<{ selected?: boolean; right?: boolean }>`
+  display: block;
+  font-size: 1.2rem;
+  height: 2.4rem;
+  padding: 0 0.8rem;
+  white-space: nowrap;
+  text-overflow: ellipsis;
```

Comment:
Dont like that much how it looks here:
![image](https://user-images.githubusercontent.com/18649057/169905519-a4fcbf9b-46e5-4de0-8930-aad1186c5a56.png)

Maybe we should display more characters by default, at least 7 imo

Reply:
Yeah, the current specs is to be 1 line for desktop, so it is tricky. I will check if I can do some flex magic

Reply:
Yeah, it will be imposible to show something nice in 1 line with 5 tokens..

Reply:
ok, lets go with that and iterate after if we get feedback